.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+*.log
+dist
```

Comment:
You'll need to add an `.npmignore` file or a `files` field in `package.json` in order for `dist/` to get published even though it's ignored in `.gitignore`.